components/RadiolabFooter.vue

```diff
@@ -1,14 +1,13 @@
 <script setup>
-import { onMounted } from 'vue'
-import menuItemsImport from '~/utilities/menuItems'
+import { ref, onMounted } from 'vue'
```

Comment:
how did this ever work without the ref import?

Reply:
and why did vscode not yell at me about it?

Reply:
it never yelled at me either! but the unit test did not like it lol